src/Microsoft.Health.Dicom.SqlServer/Features/Query/DicomTagSqlEntry.cs

```diff
@@ -26,12 +28,37 @@ internal class DicomTagSqlEntry
                 { DicomTag.SOPInstanceUID, new DicomTagSqlEntry(DicomTag.SOPInstanceUID, SqlTableType.InstanceTable, VLatest.Instance.SopInstanceUid) },
         };
 
-        private DicomTagSqlEntry(DicomTag dicomTag, SqlTableType sqlTableType, Column sqlColumn, string fullTextIndexColumnName = null)
+        private static Dictionary<DicomVR, DicomTagSqlEntry> _customTagToSqlMappingCache = new Dictionary<DicomVR, DicomTagSqlEntry>()
+        {
+                { DicomVR.DA, new DicomTagSqlEntry(null, SqlTableType.CustomTagDateTimeTable, VLatest.CustomTagDateTime.TagValue, null, VLatest.CustomTagDateTime.TagKey) },
```

Comment:
I see you are passing null to DicomTag. Do you want to change the constructor to not include dicomTag and remove that property. It is not being used. in the queryGenerator